src/blockdata/opcodes.rs

```diff
@@ -882,6 +882,15 @@ mod tests {
         }
     }
 
+    macro_rules! regression {
+        ($op:ident, $s:expr) => {
+            let op = all::$op;
+            let want = format!("\"{}\"", $s);
+            let got = ::serde_json::to_string(&op).unwrap();
+            assert_eq!(got, want);
+        }
+    }
```

Comment:
Something to consider: put each check into a separate function, so that if any of them fails, it'll be quick and obvious to find which one. Since there are many of them, I'd suggest this:

```rust
    macro_rules! regression {
        ($op:ident, $s:expr) => {
            #[test]
            #[allow(non_snake_case)]
            fn $op() {
            let op = all::$op;
            let want = format!("\"{}\"", $s);
            let got = ::serde_json::to_string(&op).unwrap();
            assert_eq!(got, want);
        }
    }
    mod ops {
        regression!(OP_PUSHBYTES_0, "OP_PUSHBYTES_0");
        // ...
    }
```

Reply:
BOOM! This is nice, thanks.

Reply:
Adding the third parameter to `assert_eq!` would achieve a similar result (quickly finding breaks) without adding too many tests?

Reply:
There's still benefit of running them independently by seeing all failures. Not sure how important it is.

Reply:
Since this is regression testing and not really expected to break very often I'm leaning towards @RCasatta's idea, less tests but still able to debug. Open to more opinions however.

Reply:
OK, still recommend using a small macro to decrease duplication.

Reply:
Most definitely, thanks.